### PR TITLE
Drabzit patch 1

### PR DESCRIPTION
**Pull request by Drabzit**
## Note to reviewers

 Updated Profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3328"></a>

### CODE REVIEW

File: data/Drabzit.json [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+	"socials": [{
+			"icon": "FaGithub",
+			"url": "https://github.com/Drabzit"
+		},
+		{
+			"icon": "FaLinkedin",
+			"url": "https:www.linkedin.com/in/christopher-bardsley-1291b0191"
+		},
+		{
+			"icon": "FaInstergram ",
```
Comment (ChinmayMhatre) [inline data/Drabzit.json:19]: Typo in the icon name.
```suggestion
			"icon": "FaInstagram ",
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]

File: data/Drabzit.json (hunk 1 above)
Comment (Drabzit) [reply data/Drabzit.json]: thanks - just found the page with all the icons - and updated it. before I saw this :D

**Review by ChinmayMhatre — APPROVED** [review]
Looks good!

File: data/Drabzit.json [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+	"socials": [{
+			"icon": "FaGithub",
+			"url": "https://github.com/Drabzit"
+		},
+		{
+			"icon": "FaLinkedin",
+			"url": "https:www.linkedin.com/in/christopher-bardsley-1291b0191"
+		},
+		{
+			"icon": "FaInstagram ",
```
Comment (schmelto) [inline data/Drabzit.json:19]:
```suggestion
			"icon": "FaInstagram",
```

File: data/Drabzit.json [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
-  "bio": "Living 1 Life at a Time, Crazy IT Dude",
-  "avatar": "https://github.com/Drabzit.png"
+	"name": "Chris Bardsley",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Living 1 Life at a Time, Crazy IT Dude",
+	"avatar": "https://github.com/Drabzit.png",
+	"tags": ["Noob", "Gamer", "Dad"],
+
+
```
Comment (schmelto) [inline data/Drabzit.json:9]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉
Would you like to add any [links](https://linkfree.eddiehub.io/docs/how-to-guides/links)?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**